pkg/server/plugin/nodeattestor/x509pop/x509pop.go

```diff
@@ -84,10 +106,22 @@ func buildConfig(coreConfig catalog.CoreConfig, hclText string, status *pluginco
 		pathTemplate = tmpl
 	}
 
+	var svidPrefix string
+	if hclConfig.SVIDPrefix == nil {
+		svidPrefix = "/spire-exchange/"
+	} else {
+		svidPrefix = *hclConfig.SVIDPrefix
+		if !strings.HasSuffix(svidPrefix, "/") {
+			svidPrefix += "/"
+		}
+	}
```

Comment:
this can be simplified:
```suggestion
	svidPrefix :=  "/spire-exchange/"
	if hclConfig.SVIDPrefix != nil {
		svidPrefix = *hclConfig.SVIDPrefix
		if !strings.HasSuffix(svidPrefix, "/") {
			svidPrefix += "/"
		}
	}
```